release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2780,3 +2780,30 @@ link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[[ocp-4-9-31]]
+=== RHBA-2022:1605 - {product-title} 4.9.31 bug fix update
+
+Issued: 2022-05-02
+
+{product-title} release 4.9.31, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1605[RHBA-2022:1605] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1604[RHBA-2022:1604] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6955866[{product-title} 4.9.31 container image list]
+
+[id="ocp-kubernetes-1-22-8-updates"]
+==== Updates from Kubernetes 1.22.8
+
+This update contains changes from Kubernetes 1.22.6 up to 1.22.8. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1226[1.22.6], link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1227[1.22.7], and link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1228[1.22.8].
+
+[id="ocp-4-9-31-bug-fixes"]
+==== Bug fixes
+* Previously, a rewrite in the Application Programming Interface (API) prevented the creation of `DualStack` services without specifying the ipFamilyPolicy. This fix alerts users of the upcoming API change, ensuring awareness of the required ipFamilyPolicy specification beforehand. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050632#[*BZ#2050632*])
+
+* Previously, the git import page failed to load due to performance improvement releases, causing the page to crash. With this fix, the git import page no longer fails to load and now runs as intended. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069621#[*BZ#2069621*])
```

Comment:
Git is capitalized based on [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#_g)

Reply:
Since it is a page it should actually be 

> **Git import** page